pkg/cosign/cosign.go

```diff
@@ -181,11 +180,23 @@ func buildCosignOptions(ctx context.Context, opts images.Options) (*cosign.Check
 		}
 	}
 
+	if opts.RekorURL == "" && opts.Key == "" {
```

Comment:
Should this be only for Keyless or all attestor types?

Reply:
When I allow it for Keyed, it gives this error
```
I0525 05:09:25.555208       1 cosign.go:61] cosign "msg"="image verification failed" "error"="no matching signatures:\nsignature not found in transparency log\n searching log query: [POST /api/v1/log/entries/retrieve][400] searchLogQueryBadRequest  &{Code:400 Message:verifying signature: invalid signature when validating ASN.1 encoded signature}\n searching log query: [POST /api/v1/log/entries/retrieve][400] searchLogQueryBadRequest  &{Code:400 Message:verifying signature: invalid signature when validating ASN.1 encoded signature}\n searching log query: [POST /api/v1/log/entries/retrieve][400] searchLogQueryBadRequest  &{Code:400 Message:verifying signature: invalid signature when validating ASN.1 encoded signature}"
```

`no matching signatures:\nsignature not found in transparency log\n searching log query: [POST /api/v1/log/entries/retrieve` - This part looks related to rekor, thats why I added it just for keyless.

Log for [keyed-basic](https://github.com/kyverno/kyverno/tree/main/test/conformance/kuttl/verifyImages/clusterpolicy/standard/keyed-basic) kuttl test - https://pastebin.com/KxMnqB5s